meta/parse.pl

```diff
@@ -284,6 +285,7 @@ sub ExtractDescription
 sai_vlan_list_t         VLAN_LIST
 sai_qos_map_list_t      QOS_MAP_LIST
 sai_tunnel_map_list_t   TUNNEL_MAP_LIST
+sai_map_list_t          MAP_LIST
```

Comment:
Still seeing one issue:

saimetadata.c:29581:38: error: 'SAI_ATTR_VALUE_TYPE_MAP_LIST' undeclared here (not in a function)
     .attrvaluetype                 = SAI_ATTR_VALUE_TYPE_MAP_LIST,
                                      ^
Makefile:69: recipe for target 'saimetadata.o' failed